staging/src/k8s.io/code-generator/cmd/validation-gen/validation.go

```diff
@@ -316,8 +326,9 @@ func (c *callTreeForType) build(t *types.Type, root bool) (*callNode, error) {
 					name = field.Type.Name.Name
 				}
 			}
+			klog.V(5).InfoS("  field", "name", name)
 
-			jsonName := "<unknown>"
+			jsonName := "<unknown-json-name>"
```

Comment:
Do you know if there is any better conventions for error handling in code called from `GenerateType` ?  I was thinking of spending some time threading error all the way back up since there are some of these "produce invalid output to report a problem" and also some panics.

Reply:
I have seen all sorts of error-handling.  This one is the worst because the string doesn't cause a compile fail, but at least it's visible in generated code.  I am open to doing better, but bubbling that error back up is tedious.

Reply:
Cool. I've got a TODO, I'll circle back later.